bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -1082,6 +1082,15 @@ int handleGotFocus(long pView, long pArg) {
 	// calls in #browserFocusIn(Event).
 	ignoreFocus = true;
 	OS.SendMessage (browser.handle, OS.WM_SETFOCUS, 0, 0);
+
+	Event newEvent = new Event();
+	newEvent.widget = browser;
+	Point position = browser.getDisplay().getCursorLocation(); // To Points
+	position = browser.getDisplay().map(null, browser, position);
+	newEvent.x = position.x; newEvent.y = position.y;
```

Comment:
nit: can you put these statements into separate lines?

Reply:
Done